File: gpu-advisory/src/ubu_gpu_advisory/noop.py
from __future__ import annotations

from .protocol import GpuAdvisoryRequest, GpuAdvisoryResponse


def advise(request: GpuAdvisoryRequest) -> GpuAdvisoryResponse:
    return {
        "request_id": request["request_id"],
        "responded_at": request["requested_at"],
        "recommendation": "manual_review",
        "rationale": "no-op advisory scaffold; no GPU hardware or torch required",
    }

File: crates/ubu-planning-advisory-protocol/src/noop.rs
use ubu_core::worker::{GpuAdvisoryRecommendation, GpuAdvisoryRequest, GpuAdvisoryResponse};

pub fn advise_noop(request: &GpuAdvisoryRequest) -> GpuAdvisoryResponse {
    GpuAdvisoryResponse {
        request_id: request.request_id.clone(),
        responded_at: request.requested_at,
        recommendation: GpuAdvisoryRecommendation::ManualReview,
        rationale: Some("no-op advisory scaffold; no GPU hardware or torch required".to_string()),
        estimated_cost: None,
    }
}
